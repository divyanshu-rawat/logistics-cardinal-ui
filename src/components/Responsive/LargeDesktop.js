import React from 'react';
import PropTypes from 'prop-types';
import { withTheme } from 'styled-components';
import Responsive from 'react-responsive';

export const LargeDesktop = ({ theme, ...props }) => (
  <Responsive minWidth={theme.breakpoints.largeDesktop.minWidth} {...props} />
);

LargeDesktop.propTypes = {
  theme: PropTypes.object.isRequired,
};

export default withTheme(LargeDesktop);
