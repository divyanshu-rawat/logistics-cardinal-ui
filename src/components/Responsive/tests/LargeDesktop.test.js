import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { LargeDesktop } from '../LargeDesktop';

import Themes from '../../../themes';

const props = {
  theme: Themes.rooster,
  someOtherProp: true,
};

describe('<LargeDesktop />', () => {
  it('should render the component', () => {
    const tree = toJson(
      shallow(<LargeDesktop {...props}>Test...</LargeDesktop>),
    );
    expect(tree).toMatchSnapshot();
  });
});
