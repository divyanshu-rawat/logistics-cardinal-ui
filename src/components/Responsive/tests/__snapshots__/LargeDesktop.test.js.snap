// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LargeDesktop /> should render the component 1`] = `
<MediaQuery
  minWidth={1200}
  someOtherProp={true}
  values={Object {}}
>
  Test...
</MediaQuery>
`;
